access-modifier-annotation/src/main/java/org/kohsuke/accmod/restrictions/DoNotUse.java

```diff
@@ -54,12 +54,16 @@ public void invoked(Location loc, RestrictedElement target, ErrorListener errorL
         error(loc,target,errorListener);
     }
 
+    public void overridden(Location loc, RestrictedElement target, ErrorListener errorListener) {
+        error(loc,target,errorListener);
```

Comment:
This might be fixing a bug in existing usages:

```java
class A {
  @Restricted(DoNotUse.class)
  void m() {}
}
class B extends A {
  @Override void m() {} // ought to fail
}
```

If so, would deserve an IT.